scripts/completions/install.py

```diff
@@ -189,6 +202,101 @@ def _update_rc(
         assert mode == "uninstall"
 
 
+def _update_conda_scripts(
+    completions_dir: pathlib.Path,
+    mode: ConfigureMode,
+) -> None:
+    """Try to add a `source /.../completions/setup.{shell}` line automatically to conda's activation scripts.
+
+    Args:
+        completions_dir: Path to location of this script.
+        mode: Install or uninstall completions.
+    """
+
+    # Install or uninstall `source_line`.
+    source_lines = _get_source_script(completions_dir, None)
+
+    conda_path = pathlib.Path(os.environ["CONDA_PREFIX"])
+    activate_path = conda_path / "etc/conda/activate.d/nerfstudio-activate.sh"
+    if mode == "uninstall":
+        if activate_path.exists():
+            os.remove(activate_path)
+        CONSOLE.log(f":broom: Existing completions uninstalled from {conda_path}.")
+    elif mode == "install":
+        # Install completions.
+        with activate_path.open("w+", encoding="utf8") as f:
+            f.write(source_lines)
+        # TODO: handle also the deactivate case
```

Comment:
in scope for this PR or no?

Reply:
It should be. Unfortunately, I don't know how to do it. Once the completions are sourced it is hard to undo it. I am not sure if it is that big of an issue.

Reply:
Yeah, I agree it's low priority.

I checked locally, and it looks like this can be done with something like
```
complete -r ns-train
```
in bash[^1], or
```
unset '_comps[ns-train]'
```
in zsh[^2].

Do you see anything wrong with this + do you have time to give implementation a try? If it looks sensible but you don't have time we could also just merge as-is and open another issue. 

[^1]: https://stackoverflow.com/questions/43794270/disable-or-unset-specific-bash-completion
[^2]: https://unix.stackexchange.com/questions/589807/how-to-remove-one-command-completion-from-zsh

Reply:
Thanks! I added the implementation. It works for bash, but I haven't tested it with zsh.

Reply:
Works great in zsh for me :)